doc/rados/configuration/osd-config-ref.rst

```diff
@@ -308,7 +308,7 @@ scrubbing operations.
 ``osd scrub chunk min``
 
 :Description: The minimal number of object store chunks to scrub during single operation.
-              Ceph blocks writes to single chunk during scrub.
```

Comment:
hmm, I think "writes" is the plural form of "write" as a noun here

Reply:
right, updated